src/services/review/RepoCheckerService.ts

```diff
@@ -375,91 +400,107 @@ export default class RepoCheckerService {
           fileName,
           viewablePageInCms,
         ] of mapOfMdFilesAndViewableLinkInCms) {
-          const file = fs.readFileSync(path.join(repoPath, fileName), "utf8")
-
-          const filePermalink = file
-            .split("---")
-            ?.at(1)
-            ?.split("permalink: ")
-            ?.at(1)
-            ?.split("\n")
-            ?.at(0)
-
-          if (!filePermalink) {
-            continue
-          }
-          const normalisedPermalink = this.normalisePermalink(filePermalink)
-
-          // we are only parsing out the front matter
-          const fileContent = file.split("---")?.slice(2).join("---")
-          if (!fileContent) {
-            continue
-          }
           findErrors.push(
```

Comment:
ok with the `push` but it's difficult to tell what's being pushed exactly.

a quick way to solve this would just be to split the actual variable and the push 

```ts
const error = ResultAsync.fromPromise(...)
fileErrors.push(error)
```

Reply:
done